pkg/smokescreen/egressacl.go

```diff
@@ -1,6 +1,6 @@
 package smokescreen
 
 type EgressAcl interface {
-	Decide(fromService string, toHost string) (EgressAclDecision, bool, error)
+	Decide(fromService string, toHost string) (EgressAclDecision, string, bool, error)
```

Comment:
I think it's worth documenting here and in the implementation function header what each of the return values represent.

Reply:
Sure.